src/schema/v2/identityVerifications.ts

```diff
@@ -0,0 +1,58 @@
+import { GraphQLFieldConfig, GraphQLInt, GraphQLString } from "graphql"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { CursorPageable, pageable } from "relay-cursor-paging"
+import { ResolverContext } from "types/graphql"
+import {
+  connectionWithCursorInfo,
+  paginationResolver,
+} from "./fields/pagination"
+import {
+  identityVerificationConnection,
+  IdentityVerificationType,
+} from "./identity_verification"
+
+type: connectionWithCursorInfo({ nodeType: IdentityVerificationType })
```

Comment:
What's this `type: ` line hanging out on its own? (i might be missing something where `graphql-js` correctly interprets?) but I think this is copy-pasta or left over from something.

Reply:
I was naïvely copying from [this doc](https://github.com/artsy/metaphysics/blob/main/docs/pagination.md#1-connection-type) and didn't understand what it did, but figured it was correct because there were no warnings. Removed!